Sources/PackageLoading/ModuleMapGeneration.swift

```diff
@@ -106,7 +106,6 @@ extension ClangModule {
         case .directory(let path):
             output += "    umbrella \"\(path)\"\n"
         }
-        output += "    link \"\(c99name)\"\n"
```

Comment:
In that case conditional behaviour is needed inside generation logic. The same conditional handle the linking and we can avoid manually linking, what do you think is better?